node-project/programs/calculateFrequency.js

```diff
@@ -0,0 +1,13 @@
+exports.calculateFrequency = function (string) {
+  var frequency={};
+  var loop;
+  for(loop=0;loop<string.length;loop++)
+    {
```

Comment:
@psinha20 be consistent when doing code indentation and always use space before and after the assignment operator also start and end of the for loop.